gradle/libs.versions.toml

```diff
@@ -83,6 +83,7 @@ androidx-compose-accompanist-placeholder = { module = "com.google.accompanist:ac
 androidx-compose-accompanist-swiperefresh = { module = "com.google.accompanist:accompanist-swiperefresh", version.ref = "accompanist" }
 androidx-compose-accompanist-systemUi = { module = "com.google.accompanist:accompanist-systemuicontroller", version.ref = "accompanist" }
 androidx-compose-animation = { module = "androidx.compose.animation:animation", version.ref = "compose-animation" }
+androidx-compose-bom = { module = "androidx.compose:compose-bom", version = "2022.12.00" }
 androidx-compose-compiler = { module = "androidx.compose.compiler:compiler", version.ref = "compose-compiler" }
```

Comment:
Won't we want to remove all the `version.ref` uses here as well? Otherwise we'll be overriding BOM version

Reply:
version catalogs still require _something_ to be defined, and in theory they can race ahead. This just gives us an extra baseline (think of it as "everything must be _at least_ this version or higher)